website/docs/cloud-docs/workspaces/settings/drift-detection.mdx

```diff
@@ -0,0 +1,53 @@
+---
+page_title: Drift Detection - Workspaces - Terraform Cloud and Terraform Enterprise
+description: >-
+  Enable automatic jobs to check whether your real-world infrastructure matches your Terraform state file.
+---
+
+# Drift Detection
+
+-> **Note:** Drift detection is in beta. Please [provide feedback](https://forms.gle/BwjgvrKQcLbgqK3f7) about how we can improve. Drift detection is available in the paid **Business** tier. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
+
+For Terraform, drift means that your real-world infrastructure no longer matches your Terraform state file. Drift can occur when other members of your organization make changes to your infrastructure outside of the official Terraform workflow. For example, a colleague may modify resource configuration directly in the cloud provider console during a production incident.
+
+> **Hands On:** Try our [Manage Resource Drift](https://learn.hashicorp.com/tutorials/terraform/resource-drift) tutorial on HashiCorp Learn to practice detecting and fixing differences between your state file and real infrastructure.
+
+When you enable drift detection in a workspace, Terraform Cloud automatically runs a drift detection assessment for that workspace every 24 hours. This involves checking the current state of your real-world infrastructure against the latest version of the workspace’s state file. This lets you monitor drift in your infrastructure without building a custom solution.
+
+## Requirements
+
+Workspaces must must have the following settings to enable drift detection:
+- Terraform version 0.15.4+
+- [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode) or [Agent execution mode](/cloud-docs/agents#configuring-workspaces-to-use-the-agent) for Terraform runs
+
+## Enable Drift Detection
+
+All workspaces in an organization that meet the [requirements](#requirements) can use drift detection. To enable drift detection within a workspace:
+1. Go to the workspace and click **Settings > General**.
+1. Select **Enable** under **Drift Detection**.
+1. Click **Save settings**.
```

Comment:
```suggestion
To enable drift detection within a workspace:
1. Verify that the workspace satisfies the [requirements](#requirements).
1. Go to the workspace and click **Settings > General**.
1. Select **Enable** under **Drift Detection**.
1. Click **Save settings**.
```

feels a little obvious that it's available to all workspaces with required settings so i moved it into the list of steps. ignore if this feels weird, i dont feel strongly about this one